### PR TITLE
Liquidity Hub / Borrow Module Interest Rate Integration

### PR DESCRIPTION
**Pull request by yan-man**
- incorporate IR (interest rate) strategy into borrow module
- resolve calculations involving IR
- apply this to basic bm credit line with a fixed IR

### CODE REVIEW

File: src/contracts/BorrowModule.sol [hunk 1]
```diff
@@ -174,12 +143,14 @@ contract BorrowModule is IBorrowModule {
...
   }
 
-  function getInterestRate() public pure returns (uint256) {
+  function getInterestRate(uint256 assetId) public pure returns (uint256) {
     // TODO: call from interest rate strategy
     return 0;
   }
 
-  function calculateInterestRates() public pure returns (uint256) {
+  function calculateInterestRates(
```
Comment (miguelmtzinf) [inline src/contracts/BorrowModule.sol]: this is actually part of the IR contract

File: src/contracts/BorrowModule.sol [hunk 2]
```diff
@@ -174,12 +143,14 @@ contract BorrowModule is IBorrowModule {
     });
   }
 
-  function getInterestRate() public pure returns (uint256) {
+  function getInterestRate(uint256 assetId) public pure returns (uint256) {
     // TODO: call from interest rate strategy
```
Comment (miguelmtzinf) [inline src/contracts/BorrowModule.sol]: nope, just read from storage

File: src/contracts/BorrowModule.sol (hunk 1 above)
Comment (yan-man) [reply src/contracts/BorrowModule.sol]: essentially just left this to convert BIPs (returned from IR contract) to ray for calcs. Should IR contract be returning rate in ray?

Comment (yan-man) [reply src/contracts/BorrowModule.sol]: resolved on call - do conversion for now in the borrow module from BIPS to ray

**Pull request merged**